wrench/src/binary_frame_reader.rs

```diff
@@ -152,7 +152,10 @@ impl WrenchThing for BinaryFrameReader {
                         }
                         _ => {}
                     }
-                    self.frame_data.push(Item::Message(msg));
```

Comment:
Let's incorporate this logic into the match statement above, to make it a bit tidier. We should also add a brief comment describing why we ignore DeleteDocument messages during replay.